pkg/cmd/server/api/validation/master.go

```diff
@@ -131,6 +135,36 @@ func ValidateProjectNodeSelector(nodeSelector string) fielderrors.ValidationErro
 	return allErrs
 }
 
+func ValidateServiceAccountConfig(config api.ServiceAccountConfig) fielderrors.ValidationErrorList {
+	allErrs := fielderrors.ValidationErrorList{}
+
+	for i, name := range config.ManagedNames {
+		if ok, msg := kvalidation.ValidateServiceAccountName(name, false); !ok {
+			allErrs = append(allErrs, fielderrors.NewFieldInvalid(fmt.Sprintf("", i), name, msg))
+		}
+	}
+
+	if len(config.PrivateKeyFile) > 0 {
```

Comment:
not if we're running against an external kube which is running the token controller